cg/constants/housekeeper_tags.py

```diff
@@ -64,8 +64,37 @@ class GensAnalysisTag:
     FRACSNP: List[str] = ["gens", "fracsnp", "bed"]
 
 
+class BalsamicProtectedTags:
+    """Balsamic pipeline protected tags by type."""
+
+    QC: List[str] = [
+        "balsamic-config",
+        "balsamic-dag",
+        "balsamic-report",
+        "delivery-report",
+        "multiqc-html",
+        "multiqc-json",
+        "qc-metrics",
+    ]
+    VARIANT_CALLERS: List[str] = [
+        "ascatngs",
+        "cnv-report",
+        "cnvkit",
+        "delly",
+        "germline",
+        "svdb",
+        "tnscope",
+        "tnscope-umi",
+        "vardict",
+        "vcf2cytosure",
+    ]
```

Comment:
I opted for a general list, but let me know if you think it's better to have more specific tags... It's not extra work as I have the lists already at hand